tests/test_r_regressions.py

```
@@ -0,0 +1,179 @@
+    def test_source_named_argument(self, tmp_path: Path):
+        """source(file = 'utils.R') should extract 'utils.R'."""
+        from tldr.cross_file_calls import parse_r_imports
+
+        r_file = tmp_path / "test.R"
+        r_file.write_text("source(file = 'utils.R')\n")
+
+        imports = parse_r_imports(r_file)
+
+        assert len(imports) == 1
+        assert imports[0]["module"] == "utils.R"
+        assert imports[0]["is_source"] == True
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `imports[0]["is_source"]:` for truth checks

Replace with `imports[0]["is_source"]`

(E712)